src/scripts/tweaks/unsticky_tab_bar.js

```diff
@@ -8,6 +8,7 @@ const styleElement = buildStyle(`
     transition: none !important;
   }
   ${keyToCss('post')} ${keyToCss('stickyContainer')} > ${keyToCss('avatar')} { top: 69px !important; }
+  ${keyToCss('post')} ${keyToCss('stickyContainer')} > ${keyToCss('avatar')}${keyToCss('newDesktopLayout')} { top: 14px !important; }
```

Comment:
Could combine these into one selector with a different approach

```suggestion
  ${keyToCss('post')} ${keyToCss('stickyContainer')} > ${keyToCss('avatar')} {
    --dashboard-tabs-header-height: 0px !important;
  }
```

Reply:
oh, damn, yeah